ansible_collections/ds/ansible_ds/playbooks/dscreate.yml

```diff
@@ -1,47 +1,41 @@
-- name: Playbook to create a new instance if it does not alreadfy exist
+---
+- name: Playbook to create a DS instance
   hosts: localhost
+
+  vars:
+    # Note: all variables values have be set to be 
+    # Instance name
+    instance: "i1"
```

Comment:
I think we need to fix the variable names here. See:
https://github.com/redhat-cop/automation-good-practices/tree/main/roles#naming-things

```
All defaults and all arguments to a role should have a name that begins with the role name to help avoid collision with other names. Avoid names like packages in favor of a name like foo_packages.

Rationale
Ansible has no namespaces, doing so reduces the potential for conflicts and makes clear what role a given variable belongs to.)
```